src/pages/token/rewards/RewardsPage.tsx

```diff
@@ -77,17 +82,20 @@ const $GridLayout = styled.div<{ showMigratePanel?: boolean }>`
     gap: var(--gap);
   }
 
+  // xccx figure out migrate
   ${({ showMigratePanel }) =>
     showMigratePanel
       ? css`
           grid-template-areas:
             'migrate migrate'
+            'chart chart'
```

Comment:
it seems like currently there's extra padding on the top of the rewards page:
![Screenshot 2024-05-28 at 4 21 40 PM](https://github.com/dydxprotocol/v4-web/assets/9400120/892955fc-953d-438d-a798-ea2f4f76de49)
vs before:
![Screenshot 2024-05-28 at 4 21 56 PM](https://github.com/dydxprotocol/v4-web/assets/9400120/76b1bf25-5d6a-433d-be8c-324336d46d9f)

Reply:
oh hmm I see yeah - that's from the `gap` (and having chart area when it's not rendered) - will fix up in follow up!